libs/hdf-converters/test/mappers/reverse/xccdf_reverse_mapper.spec.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import {FromHDFToXCCDFMapper} from '../../../index';
+
+describe('XCCDF Results Revse Mapper', () => {
+  it('Successfully converts HDF into XCCDF-Results', () => {
+    const inputData = fs.readFileSync(
+      'sample_jsons/xccdf_reverse_mapper/sample_input_report/rhel7-results.json',
+      {encoding: 'utf-8'}
+    );
+    const outputTemplate = fs.readFileSync(
+      'src/converters-from-hdf/xccdf/hdf2xccdf-results-template.xml'
+    );
+    const mapper = new FromHDFToXCCDFMapper(
+      inputData,
+      outputTemplate.toString(),
+      true
+    );
+
+    const converted = mapper.toXCCDF();
+
+    const expected = fs.readFileSync(
+      'sample_jsons/xccdf_reverse_mapper/rhel7-xccdf-results.xml',
+      'utf-8'
+    );
+
+    expect(converted).toEqual(expected);
+  });
+});
```

Comment:
Is it beneficial to add a test for the case of using this mapper with an incorrect file, e.g. a non valid HDF file?